deepspeed/__init__.py

```diff
@@ -119,6 +123,25 @@ def initialize(args=None,
 
     assert model is not None, "deepspeed.initialize requires a model"
 
+    global dist
+    from deepspeed import comm as dist
+    from deepspeed.comm import supported_torch_version
+    # This supported_torch_version check is for torch1.2 compatibility only
```

Comment:
btw, you no longer support pt-1.2, so just as well remove that old code?

Reply:
@stas00 we were discussing this internally and agree we should remove it since we don't officially support older torch versions now. Latest commit removes it.

Reply:
Spring clean up, yay! 

Thank you, Michael!